spec/controllers/idv/confirmations_controller_spec.rb

```diff
@@ -25,34 +25,34 @@
     end
 
     describe 'all questions answered correctly' do
-      it 'shows success' do
+      it 'updates profile and redirects to SP' do
         complete_idv_session(true)
 
         get :index
 
-        expect(response.status).to eq 200
-        expect(response.body).to include(t('idv.titles.complete'))
+        expect(response).to redirect_to('http://the-original-sp.example.com')
+        expect(flash[:success]).to eq(t('idv.titles.complete'))
 
         profile.reload
 
         expect(profile).to be_active
         expect(profile).to be_verified
+
+        # PII removed from session
+        expect(subject.user_session[:idv][:applicant]).to eq nil
```

Comment:
In this test I am mocking `after_sign_in_path_for` from ApplicationController by explicitly setting `session[:saml_request_url]`